app/models/candidate_interface/satisfaction_survey_form.rb

```diff
@@ -0,0 +1,26 @@
+module CandidateInterface
+  class SatisfactionSurveyForm
+    include ActiveModel::Model
+
+    attr_accessor :question, :response
+
+    validates :question, presence: true
+
+    def save(application_form)
+      return false unless valid?
+
+      remove_strongly_agree_and_strongly_disagree_text(@response)
```

Comment:
Hmm - should we actually remove this? 🤔

We could also save it with, and remove it later upon presenting if we need to.

Does this complicate potentially amending your answers?

Reply:
Actually - why do we need to do this? How come we don't get back `'1'` since that's what we pass in as the `value=` of the radio button?

Reply:
yea that's a 🤦‍♂ moment haha